mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -779,52 +779,52 @@ Feature: Loans tests
     And match accounts[0].paymentStatus.name == 'Suspended claim returned'
     And match accounts[1].paymentStatus.name == 'Suspended claim returned'
 
-  Scenario: When an existing loan is aged to lost update agedToLostDate, item status to Aged to lost
-
-    * def extItemBarcode = 'FAT-1000IBC'
-    * def extUserBarcode = 'FAT-1000UBC'
-    * def extLoanDate = '2020-01-01T00:00:00.000Z'
-
-    # location and service point setup
-    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostLocation')
-    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostServicePoint')
-
-    # post an item
-    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostInstance')
-    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostHoldings')
-    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemBarcode: #(extItemBarcode) }
-
-    # post a group and an user
-    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostGroup') { extUserGroupId: #(groupId) }
-    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserBarcode: #(extUserBarcode) }
-
-    # checkOut the item
-    * def checkOutResponse = call read('classpath:vega/mod-circulation/features/util/initData.feature@PostCheckOut') { extCheckOutUserBarcode: #(extUserBarcode), extCheckOutItemBarcode: #(extItemBarcode), extLoanDate: #(extLoanDate) }
-    * def extLoanId = checkOutResponse.response.id
-
-    # update age-to-lost processor delay time
-    Given path '/_/proxy/tenants/' + tenant + '/timers'
-    And request '{"id":"mod-circulation_6","routingEntry":{"unit": "second", "delay":"1"} }'
-    When method PATCH
-    Then status 204
-
-    # get the loan and verify that the loan has been aged to lost and got agedToLostDate
-    * configure retry = { count: 5, interval: 1000 }
-    Given path 'loan-storage', 'loans', extLoanId
-    And retry until response.itemStatus == 'Aged to lost'
-    When method GET
-    Then status 200
-    And match $.agedToLostDelayedBilling.agedToLostDate == '#present'
-    And match $.itemStatus == 'Aged to lost'
-
-    # revert retry configuration to default values
-    * configure retry = { count: 3, interval: 3000 }
-
-    # revert age-to-lost processor delay time
-    Given path '/_/proxy/tenants/' + tenant + '/timers'
-    And request '{"id":"mod-circulation_6","routingEntry":{"unit": "minute", "delay":"30"} }'
-    When method PATCH
-    Then status 204
+#  Scenario: When an existing loan is aged to lost update agedToLostDate, item status to Aged to lost
```

Comment:
I think we  can use @ignore annotation, please check if it works

Reply:
In other modules they use @ignore on Feature and @Ignore on Scenario, but when I use them Scenarios were not skipped.